.gitignore

```diff
@@ -9,7 +9,7 @@
 bld/
 bin/
 obj/
-src/packages/
+**/packages/*
```

Comment:
>* [](http://example.com/codeflow?start=12&length=1)

for people wondering:

we had:
    "\binskim\src\sarif-sdk\packages"
    "\binskim\src\packages"

now one additional:
    "\binskim\src\BuildSamples\SourceLink\packages"

I think the last wildcard is redundant.